### ISSUE TITLE
Phone field errors [with country code drop]

### ISSUE THREAD

**Issue by umkasanki** [verbb/formie#561]
**Description**
1. Then I use phone field with empty Allowed countries list all works like expected but I see some errors on country change. 

![изображение](https://user-images.githubusercontent.com/496132/131692675-21f9f9ab-49c6-4206-846c-bcd8895afbfd.png)

https://user-images.githubusercontent.com/496132/131692694-83708c61-2c73-46df-9e10-557c4ca961a6.mp4

2. If I add some countries in the Allowed Countries list I get broken dropdown and console errors

![изображение](https://user-images.githubusercontent.com/496132/131692852-28a45035-0092-4a3d-9788-bdc87831baaf.png)

https://user-images.githubusercontent.com/496132/131692875-e333df01-12b5-49b1-a950-18c932bbbe3a.mp4

**Steps to reproduce**
1. Add phone field with empty Allowed Countries list
2. Try to change country
3. Add ome countries in the Allowed Countries list
4. Reload oage

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax 
- Client-side Validation: Yes
- Custom Form Templates: Yes 

**Additional info**
- Plugin version: 1.4.14
- Craft version: 3.7.10 pro
- Multi-site: Yes 

**Additional context**
Windows / WSL2 / Nitro 2

Thanks

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.14",
  "...": "..."
}
```

Then run `composer update`.

As for your second error ("No country data for X"), that's due to constraining the available countries to pick from, but setting the default value to a country not in that collection. I've also fixed this to force-include it to prevent that error.

**Comment by umkasanki** [comment]
@engram-design Thanks for super fast support!!!
Awesome!

**Comment by engram-design** [comment]
Will just keep open to remind me to tag this in a release!

**Comment by engram-design** [comment]
Fixed in [1.4.15](https://github.com/verbb/formie/releases/tag/1.4.15)
